src/index.tsx

```diff
@@ -14,11 +15,12 @@ export type { DefaultAddress, TokenDefaults } from 'hooks/swap/useSyncTokenDefau
 export type { Theme } from 'theme'
 export { darkTheme, defaultTheme, lightTheme } from 'theme'
 
-export type SwapWidgetProps = SwapProps & WidgetProps
+export type SwapWidgetProps = HeaderProps & SwapProps & WidgetProps
 
 export function SwapWidget(props: SwapWidgetProps) {
   return (
     <Widget {...props}>
+      <Header title="Swap" {...props} />
```

Comment:
If the Header component isn't used anywhere else at the moment I think we should drop the `title` prop and just put `"Swap"` directly into the dom.